packages/core/src/http-middlewares/after/prepare-response.js

```diff
@@ -29,20 +31,23 @@ const prepareContentResponse = (resp, request) => {
   // TODO: does it make sense to not trim the signature? more equivalence to raw...
   return resp.text().then((content) => {
     // trim down the response signature a ton for simplicity
-    let json;
-    try {
-      json = JSON.parse(content);
-    } catch (err) {
-      json = undefined;
-    }
     const preppedResp = {
       status: resp.status,
-      json: json,
+      json: undefined,
+      data: undefined,
       content: content,
       request: request,
       skipThrowForStatus: request.skipThrowForStatus,
     };
     const outResp = _.extend(preppedResp, replaceHeaders(resp));
+    try {
+      if (outResp.headers.get('content-type') === FORM_TYPE) {
+        outResp.data = querystring.parse(content);
+      } else {
+        outResp.data = JSON.parse(content);
+        outResp.json = outResp.data; // DEPRECATED
+      }
+    } catch (_) {}
```

Comment:
this `_` variable shadows the root `lodash` declaration. Should probably rename to `e` or `_e` or something. I would have expected `eslint` to catch this. 🤔